pexec/managed_process.go

```diff
@@ -40,17 +37,28 @@ type ManagedProcess interface {
 // NewManagedProcess returns a new, unstarted, from the given configuration.
 func NewManagedProcess(config ProcessConfig, logger golog.Logger) ManagedProcess {
 	logger = logger.Named(fmt.Sprintf("process.%s_%s", config.ID, config.Name))
+
+	if err := config.Validate("new managed process"); err != nil {
+		logger.Warn(err)
+	}
+	// SMURF: Or better to do this then the log/warn above?
+	// if config.StopSignal == nil {
+	//	config.StopSignal = syscall.SIGTERM
+	//}
```

Comment:
Trying to make sure this doesn't break anything in direct use in RDK (where a literal constructor might result in a nil signal) Please let me know what you think is the best way/place to handle that.